Automattic/wp-calypso
# Add co-branded Jetpack & WooCommerce header during connection process

This PR builds upon the WooCommerce styles added in #32993 by updating the header to contain a co-branded WooCommerce & Jetpack logo. The header is styled per the designs at p6riRB-4rG-p2 and woocommerce/woocommerce-admin#2152. It also updates the heading and sub-heading copy.

#### Testing instructions

To Test:

* Disconnect Jetpack `Jetpack > Dashboard > Connections > Site connection > Manage site connection > Disconnect`.
* Click “Setup Jetpack”.
* On the connection/authorization screen ( https://wordpress.com/jetpack/connect/authorize?client_id= ….. ) copy and paste the whole URL.
* Replace `https://wordpress.com` with ` http://calypso.localhost:3000` and replace the `from` parameter in the URL (it may be something like `connection-banner` or `landing-page-bottom`) with  `from=woocommerce-setup-wizard` . See Pastebin `213d5`  or ping me if you have issues.
* Hit enter, and you should see the new logo and copy.
* Disconnect
* Logout
* Try the same same URL logged out
* Test the standard Jetpack connection process, without the WooCommerce query string appended, and verify you see the standard Jetpack Onboarding styles, logo, and copy.


<img width="717" alt="Screen Shot 2019-05-20 at 2 56 57 PM" src="https://user-images.githubusercontent.com/689165/58045253-ea787700-7b0f-11e9-8997-e9d9fb905234.png">

<img width="789" alt="Screen Shot 2019-05-20 at 2 57 14 PM" src="https://user-images.githubusercontent.com/689165/58045254-ea787700-7b0f-11e9-85c0-c3ac8caf3459.png">
